tachyon/math/base/big_int.h

```diff
@@ -227,6 +294,63 @@ struct ALIGNAS(internal::LimbsAlignment(N)) BigInt {
     return ret;
   }
 
+  // Constructs a BigInt value from a given array of bytes interpreted in
+  // big-endian order. The method processes each byte of the input, packs them
+  // into 64-bit limbs, and then sets these limbs in the resulting BigInt. If
+  // the system is big-endian, adjustments are made to ensure correct byte
```

Comment:
Do you assume the platform is little-endian or means the opposite case of function?

```suggestion
  // the system is little-endian, adjustments are made to ensure correct byte
```